run-system-tests/action.yml

```diff
@@ -375,15 +416,15 @@ runs:
         publish_branch: test_reports
         keep_files: true
         github_token: "${{ inputs.svc_account_secret }}"
-        publish_dir: "./tests/cli_tests/${{ env.BRANCH_DIR }}"
+        publish_dir: "./${{ env.BRANCH_DIR }}"
         destination_dir: "./${{ env.BRANCH_DIR }}"
         user_name: "github-actions[bot]"
         user_email: "github-actions[bot]@users.noreply.github.com"
 
     - name: "Wait for report deployment to complete"
       if: ${{ always() && env.TESTS_RAN == 'true' && inputs.deploy_report_page == 'true' }}
       shell: 'script --return --quiet --command "bash {0}"'
-      run: counter=0; while [ $counter -lt 5 ] && curl -o /dev/null -Isw '%{http_code}\n' "https://0chain.github.io/${{ github.event.repository.name }}/${{ env.BRANCH_DIR }}/${GITHUB_SHA}/index.html?t=$(date +%s)" | grep -v 200 > /dev/null; do sleep 10 && echo "Waiting for report page to deploy..." && ((counter=counter+1)); done
+      run: counter=0; while [ $counter -lt 5 ] && curl -o /dev/null -Isw '%{http_code}\n' "https://0chain.github.io/${{ github.event.repository.name }}/${{ env.BRANCH_DIR }}/${GITHUB_SHA}/cli/index.html?t=$(date +%s)" | grep -v 200 > /dev/null; do sleep 10 && echo "Waiting for report page to deploy..." && ((counter=counter+1)); done
```

Comment:
Would need one to wait for API report deployment too?

Reply:
Nope, once one folder is present they all are due to the way gh pages works

Reply:
Oh, cool